WooCommerce/Classes/ViewRelated/Products/Edit Product/Subscription/ProductSubscription+UI.swift

```diff
@@ -0,0 +1,113 @@
+import Yosemite
+import WooFoundation
+
+extension ProductSubscription {
```

Comment:
As a general rule, I tried to avoid creating extensions on networking entities for content that is needed in specific views.

Having a concrete type doing presentation work for the view is a bit easier to maintain in the future.

For example, if in the future, we change these texts to make a change on the subscription screens, we don't know what other parts of the code that use these extension methods we will be affecting.

Does this make sense? 
Nothing too urgent to change now, but to keep in mind!

Reply:
This definitely makes sense. One reason why I did it as an extension was to make it easier to reuse the descriptions across multiple views:

1. Simple Subscription product details
2. Simple Subscription subscription settings
3. Variable Subscription variation details
4. Variable Subscription variation subscription settings

I thought about creating a helper for this, rather than doing it in an extension, but I reasoned that these strings are integrally connected to the `ProductSubscription` type. They describe the ways that the individual `ProductSubscription` properties are related to one another, so if we changed it in one place we'd likely need to change it everywhere.

But I can see how that could get harder to maintain if we did need to make selective changes to the UI. I'll look at changing this as I continue to work on the subscription screens.

Reply:
Looping back to note that I updated this in #9498. And I already see the value in creating these descriptions separately for each view that needs them. :)